qcodes/dataset/export_config.py

```diff
@@ -0,0 +1,110 @@
+import enum
+import logging
+
+from os.path import normpath, expanduser, exists
+from typing import Union, Optional
+
+from qcodes import config
+
+_log = logging.getLogger(__name__)
+
+
+DATASET_CONFIG_SECTION = "dataset"
+EXPORT_TYPE = "export_type"
+EXPORT_PATH = "export_path"
+EXPORT_PREFIX = "export_prefix"
+
+
+class DataExportType(enum.Enum):
+    """File extensions for supported data types to export data"""
+    NETCDF = "nc"
+    CSV = "csv"
+
+
+def set_data_export_type(export_type: str) -> None:
+    """Set data export type
+
+    Args:
+        export_type: Export type to use.
+            Currently supported values: netcdf, csv.
+    """
+    # disable file export
+    if export_type is None:
+        config[DATASET_CONFIG_SECTION][EXPORT_TYPE] = None
+
+    elif hasattr(DataExportType, export_type.upper()):
+        config[DATASET_CONFIG_SECTION][EXPORT_TYPE] = export_type.upper()
+
+    else:
+        _log.warning(
```

Comment:
This passes with a warning because I don't want users to end up in a situation where they just ran an hour long measurement and then get an ugly error at the end.

Reply:
I think this is fine for now. We can make it an exception later if we find a reason to do that